rule-engines/pom.xml

```diff
@@ -17,6 +17,7 @@
         <module>easy-rules</module>
         <module>openl-tablets</module>
         <module>rulebook</module>
+        <module>evrete</module>
```

Comment:
Move this line up after easy-rules so that the modules are listed alphabetically